microcosm_flask/conventions/base.py

```diff
@@ -42,6 +45,10 @@ def response_schema(self):
     def header_func(self):
         return self[3] or identity
 
+    @property
+    def response_formats(self):
+        return self[4]
```

Comment:
Do you want an `or []` here?

Reply:
`or []`. I wanted to avoid having the "default to JSON" behavior expressed in this class, since it's just a thin wrapper around a tuple, and should be agnostic to our conventions.